docs-api/frontity-packages/features-packages/wp-comments.md

```diff
@@ -0,0 +1,226 @@
+
+| Name              | Type                                                                                                                            | Description                                                                                                     |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| **`fields`**      | [object](https://github.com/frontity/frontity/blob/c61fbe2d240fea3b7c4d9835beac3f0159e2ee24/packages/wp-comments/types.ts#L223) | Form fields with their values.                                                                                  |
+| `isSubmitting`    | boolean                                                                                                                         | The comment hasn't been received by WP yet.                                                                     |
+| `isSubmitted`     | boolean                                                                                                                         | The comment has been received.                                                                                  |
+| ` isError`        | boolean                                                                                                                         | The request has failed.                                                                                         |
+| `errorMessage`    | string                                                                                                                          | Failure reason.                                                                                                 |
+| `errorCode`       | string                                                                                                                          | The error code. Those are defined internally in the WordPress REST API. Example: `rest_comment_invalid_post_id` |
+| `errorStatusCode` | number                                                                                                                          | The HTTP status code that might have been received from the WordPress REST API.                                 |
+| **`errors`**      | [object](https://github.com/frontity/frontity/blob/c61fbe2d240fea3b7c4d9835beac3f0159e2ee24/packages/wp-comments/types.ts#L223) | The validation errors that can be returned by the WordPress REST API.                                           |
+
+##### `state.comments.forms[].fields`
+
+The following map of fields, representing the current field values that have been input in the form rendered in the given post. The content of this property is updated using the **`updateFields()`** action described later.
```

Comment:
I don't find this very clear either. I presume it's associated with the paragraph above. Maybe it needs to be made clearer that these are to do with form submissions associated with a comment form that appears on each single view of a post - assuming I'm understanding this correctly. 🤷🏻‍♂️

Reply:
As https://github.com/frontity/api-reference/pull/21#discussion_r595388867 
Let me try to improve the explanation